index.html

```diff
@@ -5,6 +5,15 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="initial-scale = 1.0, maximum-scale = 1.0">
   <title>tundra dawn - three nights out now</title>
+  <link rel="apple-touch-icon" sizes="180x180" href="/assets/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/assets/favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/assets/favicon/favicon-16x16.png">
+  <link rel="manifest" href="/assets/favicon/site.webmanifest">
+  <link rel="mask-icon" href="/assets/favicon/safari-pinned-tab.svg" color="#5bbad5">
+  <link rel="shortcut icon" href="/assets/favicon/favicon.ico">
+  <meta name="msapplication-TileColor" content="#da532c">
```

Comment:
We should update the colors for the `mask-icon` and `msapplication-TileColor`.

This is what they look like currently:

![image](https://user-images.githubusercontent.com/6766444/82132444-6817a700-97a5-11ea-8441-2b609be955d1.png)

Reply:
Nice catch, will do!

Reply:
updated in 76a17dc :)